administrator/components/com_content/src/Model/ArticlesModel.php

```diff
@@ -482,6 +484,23 @@ protected function getListQuery()
                 ->bind(':language', $language);
         }
 
+        // Filter by checked out status.
+        $checkedOut = $this->getState('filter.checked_out');
+        if ($checkedOut !== '' && $checkedOut !== null && $checkedOut !== false) {
```

Comment:
```suggestion
        $checkedOut = $this->getState('filter.checked_out');

        if (is_numeric($checkedOut)) {
```
1. Leave an empty line before the `if` for better readability.
2. Wouldn't an `is_numeric($checkedOut)` work here, too? That's less complicated than checking for null and empty string and false.

Reply:
P.S. to 2. ... and it's used elsewhere in the same method for the same purpose.